pkg/dao/repository_config_test.go

```diff
@@ -42,7 +42,7 @@ func TestRepositoryConfigSuite(t *testing.T) {
 
 func (suite *RepositoryConfigSuite) TestCreate() {
 	name := "Updated"
-	url := "http://someUrl.com"
+	url := "http://someUrl.com/"
```

Comment:
```suggestion
	url := "http://someUrl.example.com/"
```
If this test will cause a DNS lookup then please use above or http://someUrl.invalid/

Reply:
this is actually part of https://github.com/content-services/content-sources-backend/pull/121  that i included here

Reply:
included here because i'm using part of its code. Will rebase once it goes in

Reply:
I have reviewed #121 and asked for a review of the use of domain names with unknown owners.

Reply:
@jlsherrill Please rebase to pick changes from #121

Reply:
rebased!

Reply:
thank you